components/prism-birb.js

```diff
@@ -0,0 +1,25 @@
+Prism.languages.birb = Prism.languages.extend('clike', {
+	'class-name': [
+		/\b[A-Z](?:\w*[a-zA-Z]\w*)?\b/,
+    	/\b[A-Z]\w*(?=\s+\w+\s*[;,=())])/
+    ],
+	'keyword': /\b(?:assert|bool|break|case|class|const|default|double|else|enum|final|follows|for|grab|if|int|nest|next|new|noSeeb|return|static|switch|throw|var|void|while)\b/,
+	'variable': /[a-z_]\w*/,
```

Comment:
> Please move this to the end of the object. It's misleading otherwise.

I have moved the items above lower to according to the positions of the overwritten tokens, but im not quite sure if this is what you suggested. Please let me know if it doesn't look right